### PR TITLE
Modularlize GQL request strings

### PR DESCRIPTION
**Pull request by gaoxk**
## Implementation description 
* move all gql request strings to `src/APIClients/mutations.ts` and `src/APIClients/queries.ts`

## Steps to test
1. poke around frontend and make sure nothing is broke 💀 

## What should reviewers focus on?
* thoughts on the organization of the files? open to splitting it into 
  * `auth_mutations.ts` and `story_queries.ts` (etc)
  * eliminating mutation/query distinction and just going with `auth_requests.ts` and `story_requests.ts`

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by andrewyguo** [comment]
Regarding 

> thoughts on the organization of the files? open to splitting it into
> auth_mutations.ts and story_queries.ts (etc)
> eliminating mutation/query distinction and just going with auth_requests.ts and story_requests.ts

Personally prefer splitting to auth_mutations.ts and story_queries.ts (etc).

File: frontend/src/APIClients/queries.ts [hunk 1]
```diff
@@ -1,3 +1,115 @@
-// TODO: move query gql strings here
+import { DocumentNode, gql } from "@apollo/client";
 
-export {};
+// Stories
+export const GET_STORY_CONTENTS = (id: number) =>
+  gql`
+      query {
```
Comment (e-wai) [inline frontend/src/APIClients/queries.ts:6]: Some of the queries are named and others are anonymous (sorry, I suspect half of these are me), apollo docs seem to prefer named - can we make this consistent?

File: frontend/src/APIClients/mutations.ts [hunk 2]
```diff
@@ -71,5 +71,70 @@ export interface ResetPasswordResponse {
...
+    $storyTranslationData: CreateStoryTranslationRequestDTO!
+  ) {
+    createStoryTranslation(storyTranslationData: $storyTranslationData) {
+      story {
+        id
+      }
+    }
+  }
+`;
+export type CreateTranslation = {
```
Comment (e-wai) [inline frontend/src/APIClients/mutations.ts:124]: Is there a good practice around what to name the response objects? Wondering if something like `CreateTranslationResponseDTO` is clearer

Comment (gaoxk) [reply frontend/src/APIClients/mutations.ts:124]: going with `CreateTranslationResponse` as per patterns in `EntityAPIClient.ts`

**Review by e-wai — APPROVED** [review]
> thoughts on the organization of the files? open to splitting it into
> auth_mutations.ts and story_queries.ts (etc)
> eliminating mutation/query distinction and just going with auth_requests.ts and story_requests.ts

I'm honestly not that opposed to everything being in the same file, not unlike the `schema.py` file, but this works too

**Comment by gaoxk** [comment]
TFTR!

**Pull request merged**